src/api/app/controllers/webui/package_controller.rb

```diff
@@ -45,6 +47,8 @@ class Webui::PackageController < Webui::WebuiController
 
   prepend_before_action :lockout_spiders, only: [:revisions, :dependency, :rdiff, :binary, :binaries, :requests, :binary_download]
 
+  after_action :verify_authorized, only: [:remove_file, :save_file, :abort_build, :trigger_rebuild, :wipe_binaries, :save_meta]
```

Comment:
I would use `except: [:branch]`, then this will also be done for new actions in the future and you force people to think about this.